### ISSUE TITLE
API reports owncloud users

### ISSUE THREAD

**Issue by eglia** [fcturner/passwords#280]
On the newest git version, the API response to fetch all passwords includes entries for the owncloud users:

`{"id":0,"user_id":"My Name","loginname":null,"website":"ownCloudUsername","address":null,"pass":false,"properties":false,"notes":false,"creation_date":null,"deleted":false},`

App version: 20-beta5
owncloud version: nextcloud 10.0.1

**Comment by fcturner** [comment]
It has always been like this, since I implemented sharing. The database listeners that ownCloud created, wouldn't let me get a list of users from another table as separate call. As there is no documentation available for developers (except for a small, almost worthless tutorial) this was the best way I could accomplish this. 

The entries that appear with an id of 0, are users you're eligible to share with. Maybe I can change the API so that those entries are excluded on being called.

**Comment by eglia** [comment]
If you could filter them out that would be great, imho those entries do not belong in the fetch all passwords request (maybe in a dedicated request?).

**Comment by fcturner** [comment]
Yeah, that dedicated request is the point. But I'll look into it!

**Comment by fcturner** [comment]
Work in progress :smile:
